envision/client.py

```diff
@@ -76,8 +76,10 @@ def __init__(
         wait_between_retries: float = 0.5,
         output_dir: str = None,
         sim_name: str = None,
+        headless: bool = False,
     ):
         self._log = logging.getLogger(self.__class__.__name__)
+        self.headless = headless
```

Comment:
I think this should be private at least for now since it looks like state change afterwards can break the object.
```suggestion
        self._headless = headless
```